src/test/kotlin/uk/gov/justice/digital/hmpps/prisonercontactregistry/integration/mock/PrisonApiMockServer.kt

```diff
@@ -114,6 +114,97 @@ class PrisonApiMockServer : WireMockServer(8092) {
     )
   }
 
+  fun stubGetOffenderMultipleContacts(offenderNo: String) {
+    stubFor(
+      get("/api/offenders/$offenderNo/contacts?approvedVisitorsOnly=true")
+        .willReturn(
+          aResponse()
+            .withHeader("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+            .withStatus(200)
+            .withBody(
+              """
+                {
+                "offenderContacts": [
```

Comment:
I know that we have done this in this project but try and not use JSON... it's an old project before my time

```
  private fun getJsonString(obj: Any): String {
    return objectMapper.writer().writeValueAsString(obj)
  }

 responseBuilder.withStatus(HttpStatus.OK.value())
            .withBody(getJsonString(visitDto))
```

Reply:
As discussed, we'll need a tech debt ticket for this. It should be done, but in it's own branch